src/main/scala/com/gu/editorialproductionmetricsmodels/models/EventTypes.scala

```diff
@@ -12,7 +12,10 @@ case class CapiData(
    firstPublicationDate: String,
    commissioningDesk: String,
    originatingSystem: OriginatingSystem,
-   productionOffice: Option[ProductionOffice])
+   productionOffice: Option[ProductionOffice],
+   commissionedWordCount: Option[Int],
+   path: String,
+   wordCount: Option[Int])
```

Comment:
Why is `wordCount` an option? Would it not always have one or does it only get written on first save in composer?

Reply:
Have discussed this with Reetta. The word count is an option in our database as it is a field that has been added retrospectively so not all entries have it. It is also optional from in the capi scala client so I think leaving it as optional makes sense.